cmd/version.go

```diff
@@ -20,11 +26,60 @@ var versionCmd = &cobra.Command{
 To see if there is a more recent version available, call the command with the
 --latest flag.
 	`,
+
 	Run: func(cmd *cobra.Command, args []string) {
```

Comment:
Let's leave it for now, but let's also reconsider what we think the convention for the codebase should be.